tests/test_postgen_dependencies.py

```diff
@@ -107,15 +122,51 @@ def test_getfixturevalue(request, factoryboy_request: Request):
     foo = request.getfixturevalue("foo")
     assert not factoryboy_request.deferred
     assert foo.value == 1
+    assert foo.secret == "super secret"
+    assert foo.number == 987
+
+
+def test_postgenerationmethodcall_getfixturevalue(request, factoryboy_request):
+    """Test default fixture value generated for ``PostGenerationMethodCall``."""
+    secret = request.getfixturevalue("foo__secret")
+    number = request.getfixturevalue("foo__number")
+    assert not factoryboy_request.deferred
+    assert secret == "super secret"
+    assert number is NotProvided
+
+
+def test_postgeneration_getfixturevalue(request, factoryboy_request):
+    """Ensure default fixture value generated for ``PostGeneration`` is `None`."""
+    set1 = request.getfixturevalue("foo__set1")
+    set2 = request.getfixturevalue("foo__set2")
+    assert not factoryboy_request.deferred
+    assert set1 is None
+    assert set2 is None
 
 
 def test_after_postgeneration(foo: Foo):
     """Test _after_postgeneration is called."""
     assert foo._create is True
-
-    foo._postgeneration_results["set1"] == "set to 1"
-    foo._postgeneration_results["baz"].foo is foo
-    assert len(foo._postgeneration_results) == 2
+    assert foo._postgeneration_results.pop("baz")
```

Comment:
We used to assert that `... is foo`

Reply:
nevermind, we didn't. I added a test for it and marked as xfail, since it's not working at the moment